enterprise-suite/es-monitor-api/prometheus.yml

```diff
@@ -13,12 +13,9 @@ rule_files:
 
 alerting:
   alertmanagers:
-  - kubernetes_sd_configs:
-    - role: pod
-    relabel_configs:
-    - source_labels: [__meta_kubernetes_namespace,__meta_kubernetes_pod_label_app,__meta_kubernetes_pod_label_component,__meta_kubernetes_pod_container_port_number]
-      regex: lightbend;prometheus;alertmanager;[0-9]+
-      action: keep
+  - static_configs:
+    - targets:
+      - {{ .AlertmanagerAddress }}
```

Comment:
I wonder if we need to ever support multiple alert managers. How does HA alertmanager work? Do you need to provide the address of each alertmanager? In which case, we probably need to support passing in multiple values.

Reply:
Yes, HA alertmanagers work by providing multiple addresses to prometheus.

Reply:
Now works with HA alertmanager clusters